docs/content/posts/2025-05-28-release-kubernetes-operator-1.12.0.md

```diff
@@ -0,0 +1,61 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.12.0 Release Announcement"
+date: "2025-05-28T18:00:00.000Z"
+authors:
+- gyfora:
+  name: "Gabor Somogyi"
+aliases:
+- /news/2025/05/28/release-kubernetes-operator-1.12.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.12.0!
+The version brings a number of important fixes and improvements to both core and autoscaler modules.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### Enhanced Error Visibility and Event Reporting
+
+- **Improved Error Transparency**: The operator now exposes detailed error messages for deployment issues, aiding in faster diagnosis and resolution.
+- **Kubernetes Event Integration**: Failures such as job transitions to the FAILED state and job restarts now generate Kubernetes events, providing better observability and alerting capabilities.
```

Comment:
This is really the general theme of the release and probably the most important improvements for the users. We have improved event reporting in the entire stack, from operator controller errors, to job startup/k8s error detection/reporting all the way to events now triggered on job exceptions.

This is just a note in addition to what you have already written, maybe we can add more details/focus on these.

Reply:
+1 to add a bit more detail on what exactly has been added / introduced int his release

Reply:
Added a more detailed description.